source/common/modules/markdown-editor/parser/markdown-parser.ts

```diff
@@ -232,7 +232,12 @@ export default function markdownParser (config?: MarkdownParserConfig): Language
         // spans of plain text.
         { name: 'HighlightContent', style: { 'HighlightContent/...': customTags.HighlightContent } },
         { name: 'Footnote', style: customTags.Footnote },
-        { name: 'FootnoteRef', style: customTags.FootnoteRef },
+        {
+          name: 'FootnoteRef',
+          style: customTags.FootnoteRef,
```

Comment:
See the `HighlightContent` implementation for a better way that hopefully does not require a new renderer.